lib/app.go

```diff
@@ -95,11 +96,6 @@ func newApp(ra *schema.RuntimeApp, podManifest *schema.PodManifest, pod *pkgPod.
 		})
 	}
 
-	// Generate annotations.
-	for _, anno := range ra.Annotations {
-		app.Annotations[anno.Name.String()] = anno.Value
```

Comment:
The change was introduced in https://github.com/coreos/rkt/commit/348facde19c8405d4dc3a0b8563fa15b7ca3c242, where `App.Annotations` was renamed to `App.CRIAnnotations`, and later to `App.UserAnnotations`.

We are setting them directly without iteration now here https://github.com/s-urbaniak/rkt/blob/crisync/lib/app.go#L64